src/Appwrite/Specification/Format/OpenAPI3.php

```diff
@@ -345,6 +345,8 @@ public function parse(): array
                     case 'Appwrite\Utopia\Database\Validator\Queries\Collections':
                     case 'Appwrite\Utopia\Database\Validator\Queries\Databases':
                     case 'Appwrite\Utopia\Database\Validator\Queries\Deployments':
+                    case 'Appwrite\Utopia\Database\Validator\Queries\Installations':
+                    case 'Appwrite\Utopia\Database\Validator\Queries\Documents':
```

Comment:
@Meldiron we need to add the other validators here for the new endpoints that've been added.